pages/index.js

```diff
@@ -1,18 +1,42 @@
 import styled from "styled-components";
-import QuestionForm from "../components/QuestionForm";
 
-export default function Home({ morningAnswers, onAddMorningAnswer }) {
+export default function Home() {
+  const week = [
+    "Sonntag",
+    "Montag",
+    "Dienstag",
+    "Mittwoch",
+    "Donnerstag",
+    "Freitag",
+    "Samstag",
+  ];
+
+  const month = [
+    "Januar",
+    "Februar",
+    "März",
+    "April",
+    "Mai",
+    "Juni",
+    "Juli",
+    "August",
+    "September",
+    "Oktober",
+    "November",
+    "Dezember",
+  ];
+  const current = new Date();
+
+  const date = `${week[current.getDay()]}, ${current.getDate()}. ${
+    month[current.getMonth()]
+  }`;
```

Comment:
If you just want to display the date, you could also use `.toLocaleString()` ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString)) to format the date, e.g.:

```suggestion
 const date = new Date().toLocaleString("de-DE", {weekday: 'long', day: 'numeric', month: 'long' });
```